src/Microsoft.Data.SqlClient/tests/tools/TDS/TDS.EndPoint/TDSClientParser.cs

```diff
@@ -54,8 +56,10 @@ public void Login()
                             // Check the client context for encryption state
                             if (Client.Context.Encryption != TDSEncryptionType.Off)
                             {
-                                // Turn on encryption before sending the response								
-                                EnableClientTransportEncryption(Client.Context.ServerHost);
```

Comment:
Does this mean if we want to use encryption we need to use certificate auth?

Reply:
No we just want to force our certificate if we have it, otherwise it will be random.